### PR TITLE
enable_vboost

### PR DESCRIPTION
**Pull request by malay-nagda**
set vboost=1 on all gpu's
Run it as a slurm job step outside container since it needs sudo 
Control it by turning `enable_vboost` flag in `config.yaml`.

### CODE REVIEW

**Comment by dimapihtar** [comment]
@malay-nagda does this param work for all slurm clusters? maybe it's better to set this params to False by default?

**Comment by malay-nagda** [comment]
> @malay-nagda does this param work for all slurm clusters? maybe it's better to set this params to False by default?
Updated.
Tested only on eos. But, makes sense to set False default.

**Comment by dimapihtar** [comment]
> > @malay-nagda does this param work for all slurm clusters? maybe it's better to set this params to False by default?
> > @dimapihtar
> > Updated.
> > Tested only on eos. But, makes sense to set False default.

could you please fix the unit tests and code style with black==19.10b0?

File: launcher_scripts/conf/config.yaml [hunk 1]
```diff
@@ -57,6 +57,8 @@ wandb_api_bcp_secret_key: null  # For BCP clusters, read the W&B api key directl
 
 bcp_no_redirect: True  # If True, all stdout and stderr will not be redirected and appear in the standard logs. If False, all stdout and stderr output will be redirected to individual files on a per-rank basis. Ignored for non-BCP clusters.
 
+enable_vboost: False # If True, enables video boost slider- benefits compute bound jobs
```
Comment (erhoo82) [inline launcher_scripts/conf/config.yaml:60]: Let's update the comment as "If True, rebalance the power budget between the compute engine and the memory such that the compute engines has greater power allocation"

**Review by erhoo82 — APPROVED** [review]

**Pull request merged**